nodejs/week1/homework/avg.js

```diff
@@ -0,0 +1,28 @@
+// "use strict";
+
+// this works  node avg.js -i 2  -n 4
+// const args = require("minimist")(process.argv.slice(2));
+
+// console.log(parseInt(args.i) + parseInt(args.n));
+
+const args = process.argv.slice(2);
+for (let i = 0; i < args.length; i++) {
+  if (typeof args[i] === "string") {
```

Comment:
What if it's not a string but for example a boolean? (see [the possible JavaScript types](https://www.w3schools.com/js/js_typeof.asp))

Usually when dealing with types, because it can be mission critical for the rest of the program to run, it is better to check if the value is of the _right_ type rather than checking if it is not of the wrong type. I.e., we should make sure all the types are numbers and only if so, continue.

For that, you can use the isNaN function.

Reply:
done